mono/metadata/threadpool-worker-default.c

```diff
@@ -221,6 +221,7 @@ rand_next (gpointer *handle, guint32 min, guint32 max)
 static void
 destroy (gpointer data)
 {
+#if 0
```

Comment:
Even a comment like "see PR 4597" would help, if PR 4597 explains what is happening